tests/shared/core/test_domain.py

```diff
@@ -779,16 +771,6 @@ def test_check_domain_sanity_on_invalid_domain():
             forms={},
         )
 
-    with pytest.raises(InvalidDomain):
-        Domain(
-            intents={},
-            entities=[],
-            slots=[],
-            responses={},
-            action_names=[],
-            forms=["random_name", "random_name"],
```

Comment:
Why was this test removed - shouldn't we test that forms as `list` raises `InvalidDomain`?

Reply:
I dropped it as it's covered by the schema. Can re-add it though.

Reply:
re-added